<launch>
	<!-- SETUP OF THE NODES AND CONFIGURATIONS -->
	
	<!-- Helmsmann controller -->
	<node name="roburoc4_trajectory_controller" pkg="roburoc4" type="roburoc4_trajectory_controller" />

</launch>
